app/models/user_bet.rb

```diff
@@ -2,4 +2,6 @@ class UserBet < ApplicationRecord
   belongs_to :user
   belongs_to :bet
   belongs_to :bet_option
+
+  scope :winners, -> { joins([:user, :bet_option]).merge(BetOption.winners) }
```

Comment:
Un-needed array here. `joins` accepts a splat, which means you can pass in aguments directly without wrapping them in an array:

`joins(:user, :bet_option)` 

http://jacopretorius.net/2012/01/splat-operator-in-ruby.html